static/sass/partials/_form.scss

```diff
@@ -0,0 +1,34 @@
+/* SASS file for form. */
+form {
+  font-family: $text-font;
+  .form-wrapper {
+    padding-left: 0;
+    padding-right: 0;
+    .col-xs-8,
+    .col-xs-4 {
+      padding-left: 0;
+      padding-right: 0;
+    }
+    .col-xs-8 {
+      text-align: left;
+    }
+    .input-text {
+      font-size: $base-font-size;
+      border: 1px solid $secondary-color;
+      padding: 6px;
+      width: 95%;
+    }
+    .submit-btn {
+      font-size: $base-font-size;
+      background: #b47b31;
+      color: $primary-bg;
+      border: none;
+      padding: 7px;
+      width: 100%;
+    }
+    .submit-btn:hover {
+      background-color: #b07337;
```

Comment:
Use variables for '#b07337', if variable does not exist, create one. You can call this variable $hover.